source/release_notes/5.0.9-8.rst

```diff
@@ -0,0 +1,37 @@
+.. _PSMDB-5.0.9-8:
+
+================================================================================
+*Percona Server for MongoDB* 5.0.9-8 (2022-06-15)
+================================================================================
+
+:Date: June 15, 2022
+:Installation: :ref:`install`
+
+Percona Server for MongoDB 5.0.9-8 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.9 Community Edition.
+It supports MongoDB 5.0.9 protocols and drivers.
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
+* The support of encrypted KMIP client certificates and keys through the password setting. 
+
+The bug fixes, provided by MongoDB and included in Percona Server for MongoDB, are the following:
+
+* `SERVER-65636 <https://jira.mongodb.org/browse/SERVER-65636>`_ - Fixed the limits of LDAP connections per host from total number of connections to the number of in-progress connections.
+* `SERVER-65137 <https://jira.mongodb.org/browse/SERVER-65137>`_ - Detect namespace changes when refreshing Collection after yielding to maintain data consistency and avoid stale catalogs.
+* `SERVER-64822 <https://jira.mongodb.org/browse/SERVER-64822>`_ - Fixed the issue with releasing the critical state too early when sharding an empty collection. This could result in unwanted writes to that collection.
+* `WT-9096 <https://jira.mongodb.org/browse/WT-9096>`_ - Fixed the wrong key/value returning during search near when the key doesn't exist
+  
+Find the full list of changes in the `MongoDB 5.0.9 Community Edition release notes <https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.9---may-31--2022>`_.
+
+Improvements
+================================================================================
+
+* :jirabug:`PSMDB-1045`: Add support for several KMIP servers 
+* :jirabug:`PSMDB-1043`: The ``kmipClientCertificateFile`` option now includes both the client private key and public certificate
```

Comment:
The file specified by kmipClientCertificateFile now includes both the client private key and public certificate. kmipClientKeyFile removed.

Reply:
Amended